patches/tModLoader/Terraria/ModLoader/ItemLoader.cs

```diff
@@ -1244,26 +1244,31 @@ private class HookList
 		/// Returns the wing item that the player is functionally using. If player.wingsLogic has been modified, so no equipped wing can be found to match what the player is using, this creates a new Item object to return.
 		/// </summary>
 		public static Item GetWing(Player player) {
```

Comment:
I'll double check it does nothing otherwise in the player.cs, but yeah, when looking at it, it's very redundant and overly-complicated. 
I'll look to remove it independent of design path.

Reply:
I haven't removed this method as it is used in horizontal and vertical wing speed fetches in ItemLoader. For now, I don't touch ItemLoader.cs, so I don't think I'll tackle that here.

Reply:
Resolved by storing "equippedWings" on the player at same time as setting winglogic.